build.sh

```diff
@@ -0,0 +1,11 @@
+git submodule update -f --init --recursive
+
+rm -R data/*
```

Comment:
What exactly is the goal of removing the content of the `data` directory. If the goal is to copy the docs from the cue repository I think it would be better to add a `vendor` directory where a git submodule for the `cuelang/cue` repositor resides and than create a symlink to the `data` directory from their.

Reply:
Good catch. This line should not have been there yet and the approach will be different anyway. This is just to set up the build command in a script.